apps/baseline/tests/test_viewsets.py

```diff
@@ -293,6 +294,25 @@ def test_filter_by_country(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(json.loads(response.content.decode("utf-8"))), 1)
 
+    def test_pagination_for_browsable_api(self):
+        LivelihoodZoneFactory.create_batch(100)
+        response = self.client.get(self.url, {"format": "json"})
+        self.assertEqual(response.status_code, 200)
+        # Check that the response contains all 100 items (no pagination for JSON)
+        self.assertEqual(len(response.data), 100 + self.num_records)
+
+        response = self.client.get(self.url, {"format": "api"})
+        self.assertEqual(response.status_code, 200)
+        soup = BeautifulSoup(response.content, "html.parser")
+        json_section = soup.find("pre", class_="content-holder")
+        # Extract text and clean HTML tags to look for the JSON data inside
+        raw_json_text = json_section.get_text()
+        soup_cleaned = BeautifulSoup(raw_json_text, "html.parser")
+        cleaned_json_text = soup_cleaned.get_text()
+        json_data = json.loads(cleaned_json_text)
+        # Check that the paginated response contains only 50 items
+        self.assertEqual(len(json_data["results"]), 50)
+
```

Comment:
@girumb please can we add another unit test that tests that we can get a paginated response on a JSON format API call by passing either the `?page=` or `?page_size=` parameters explicitly.